docs/modules/cs/index.ts.md

```diff
@@ -77,7 +77,7 @@ public addConnectionListener(listener: ConnectionListener)
 **Signature**
 
 ```ts
-public close(): Promise<void>
+public async close(): Promise<void>
```

Comment:
trocar aqui para async não vai implicar em ter que fazer modificações no  nosso backend também? Não vai precisar dar um await onde isso for chamado?

Reply:
Opa, não modifiquei nada não kkkk, isso é gerado automaticamente.

Reply:
A última pessoa que alterou esqueceu de dar yarn build

Reply:
0k

Reply:
vi que o tipo de retorno já era uma promise